.circleci/test-deploy.yml

```
@@ -139,6 +139,18 @@ jobs:
           working_directory: ~/project/sample_poetry
           command: |-
             poetry run pytest
+  uv-test:
+    executor: python/default
+    steps:
+      - checkout
+      - python/install-packages:
+          app-dir: ~/project/sample_uv
+          cache-version: << pipeline.parameters.cache-version >>
+          pkg-manager: uv
+      - run:
+          working_directory: ~/project/sample_uv
+          command: |-
+            uvx run pytest
```

```suggestion
            uv run pytest
```

To quote documentation on `uv tool run`, which `uvx` is an alias for:

> Packages are installed into an ephemeral virtual environment in the uv cache directory.

In my case, `uvx run pytest` fails to read configuration from pyproject.toml. `uv run pytest` works perfectly, however.

---

The tests passed with this suggestion.